### ISSUE TITLE
Removing all schema directories from the GDK Tools Configuration causes hard crashes.

### ISSUE THREAD

**Issue by jamiebrynes7** [spatialos/gdk-for-unity#955]
**Affects:** Release v0.1.5 and up
**Internal Ticket:** [UTY-2044](https://improbableio.atlassian.net/browse/UTY-2044)

---

#### Description

Removing all schema source directories from the GDK Tools Configuration causes hard crashes on startup and during code generation.

#### Workaround

If you get yourself into this state, you can manually edit the JSON file that is found at:

```
workers/<your-unity-project>/Assets/Config/GdkToolsConfiguration.json
```

such that there is at least 1 element in the `SchemaSourceDirs` list.

For example:

```json
    
{
    "SchemaStdLibDir": "build/dependencies/schema/standard_library",
    "SchemaSourceDirs": [
        "Assets/.Schema"
    ],
    "CodegenOutputDir": "Assets/Generated/Source"
}
```

**Comment by jamiebrynes7** [comment]
Will be fixed by https://github.com/spatialos/gdk-for-unity/pull/953 and released in v0.2.3

**Comment by jamiebrynes7** [comment]
`v0.2.3` has been released which contains a fix for this - https://github.com/spatialos/gdk-for-unity/releases/tag/0.2.3
